sections/api/typescript.md

```diff
@@ -157,11 +157,12 @@ import styled from 'styled-components'
 import Header, { Props as HeaderProps } from './Header'
 
 const Title =
-  (styled < { isActive: boolean }) &
-  (HeaderProps >
-    (({ isActive, ...rest }) => <Header {...rest} />)`
-  color: ${props => (props.isActive ? props.theme.primaryColor : props.theme.secondaryColor)}
-`)
+  styled <{ isActive: boolean } & HeaderProps>(
+    ({ isActive, ...rest }) => <Header {...rest} />
+  )`
+    color: ${(props) =>
+      props.isActive ? props.theme.primaryColor : props.theme.secondaryColor};
+`;
 ```
```

Comment:
this particular example should work with even simpler syntax:

```ts
const Title = styled(Header)<{ isActive: boolean }>`
  color: ${(props) =>
    props.isActive ? props.theme.primaryColor : props.theme.secondaryColor};
`
```

Reply:
This isn't the same thing though -- the example is trying to extract `isActive` from the props that are consumed by `Header`. In your example, `isActive` will still be a prop on `Header`